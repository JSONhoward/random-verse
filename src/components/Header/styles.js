import styled from 'styled-components'

export const HeaderStyled = styled.header`
display: flex;
align-items: center;
height: 4rem;
width: 100vw;
background-color: white;
box-shadow: 0px 1px 5px grey;
padding-left: 2rem;

@media (max-width: 500px) {
    padding-left: 0;
    justify-content: center;
}
`

export const Logo = styled.div`
font-size: x-large;
font-weight: 500;
`