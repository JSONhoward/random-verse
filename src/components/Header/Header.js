import React from 'react'

import { HeaderStyled, Logo } from './styles'

const Header = () => {
    return (
        <HeaderStyled>
            <Logo>Random Verse</Logo>
        </HeaderStyled>
    )
}

export default Header
