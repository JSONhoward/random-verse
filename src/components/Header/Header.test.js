import React from 'react'
import {create} from 'react-test-renderer'
import {screen, render} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Header from './Header'

describe('Header Tests', () => {
    test('Snapshot Test', () => {
        const tree = create(<Header />).toJSON()

        expect(tree).toMatchSnapshot()
    })

    test('Renders Logo text', () => {
        render(<Header />)

        expect(screen.getByText('Random Verse')).toBeInTheDocument()
    })
})