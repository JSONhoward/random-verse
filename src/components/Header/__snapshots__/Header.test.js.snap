// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests Snapshot Test 1`] = `
<header
  className="sc-AxjAm gIlwkJ"
>
  <div
    className="sc-AxirZ fptRDo"
  >
    Random Verse
  </div>
</header>
`;
